streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -959,28 +992,23 @@ private void maybeRecordE2ELatency(final long recordTimestamp, final long now, f
         }
     }
 
-    /**
-     * Request committing the current task's state
-     */
-    void requestCommit() {
-        commitRequested = true;
+    public InternalProcessorContext processorContext() {
+        return processorContext;
     }
 
-    /**
-     * Whether or not a request has been made to commit the current state
-     */
-    @Override
-    public boolean commitRequested() {
-        return commitRequested;
+    public boolean hasRecordsQueued() {
+        return numBuffered() > 0;
     }
 
+    // visible for testing
```

Comment:
Can we avoid these comments? I've come across too many cases where it had become untrue recently, to the point where it just seems pointless to have them.

I know you're preserving the comment that was there before while moving the method, but I think we should just clean up the comments as well.